pod/perldata.pod

```diff
@@ -57,9 +57,10 @@ which works much as the word "these" or "those" does in English,
 in that it indicates multiple values are expected.
 X<array>
 
-    @days		# ($days[0], $days[1],... $days[n])
-    @days[3,4,5]	# same as ($days[3],$days[4],$days[5])
-    @days{'a','c'}	# same as ($days{'a'},$days{'c'})
+    @days		# ($days[0], $days[1], ... $days[n])
```

Comment:
Please align these comments with the one below.

Reply:
> Please align these comments with the one below.

I would love to.
But it would be best if you did it for me.
Else, it would tangle up GitHub, in my experience.

Reply:
All that's really needed is a new commit aligning them. If it's messed up in Github, closing and re-opening with a fix would be just fine.

Reply:
To tell you the awful truth, I don't really know what you guys are talking about.
I see a weird diff above. Three red lines and one green.
So maybe those are the comments you want me to align. But maybe I will align them wrong.
Anyway, therefore, I, hereby, authorize you fellows, to align the comments as you please!
(Actually I was going to even throw in another line, showing folks even one more level of depth...)

Reply:
And in fact, all deeper levels require the @{}, but, it seems, one cannot use @{} through and through! As the shallower levels, like the one in the shown before my edit, cannot be made to use it, I think.

Reply:
<img width="696" alt="Screen Shot 2021-05-28 at 11 56 08 AM" src="https://user-images.githubusercontent.com/38994/120017660-c1670880-bfab-11eb-82e0-631980e4f38d.png">

As shown here, the start of the end of line comments are not aligned where they all were previously.  Please re-align them.